src/base/hooks/useMouseDragScroll.tsx

```diff
@@ -37,7 +37,7 @@ export const useMouseDragScroll = (
     const previousClickPosY = useRef<Positions>([0, 0, 0]);
     const previousClickTime = useRef<ClickTimes>([0, 0, 0]);
     const scrollAtT0 = useRef<[ScrollLeft: number, ScrollTop: number]>([0, 0]);
-    const inertiaTimeInterval = useRef<NodeJS.Timeout>(undefined);
+    const inertiaTimeInterval = useRef<NodeJS.Timeout | number>(undefined);
```

Comment:
```suggestion
    const inertiaTimeInterval = useRef<number>(undefined);
```

`NodeJS.Timeout` is not possible, since the  `setTimeout` usage was removed